UKHomeOffice/cerberus-service
# COP-11890 - Field level highlighting

## Description
This PR adds field level highlighting to the task details page across task versions.

### Supplemental
This PR also adds displaying the number of changes between versions.

## To Test
- Pull & run against dev
- Post a task
- Update the task and repost it again.
- Check the same task in task details, fields that are changed or that are associated with a change will be highlighted.
